### ISSUE TITLE
Docker: MongoDB doesn't start if /var/lib/mongodb is mounted (-v)

### ISSUE THREAD

**Issue by fnkr** [Countly/countly-server#239]
`/var/lib/mongodb` is owned by root then and MongoDB won't start until `chown mongodb: /var/lib/mongodb` is done manually via docker exec.

```
docker run -d -P -v /var/data/countly/mongodb:/var/lib/mongodb --name countly countly/countly-server
```

**Comment by gorkem-cetin** [comment]
@iartem I remember you made a commit but doest that fix this?

**Comment by waiterZen** [comment]
https://hub.docker.com/r/countly/countly-server/builds/bplqkdawbbnzm7uth5qajmb/ docker build got error

**Comment by waiterZen** [comment]
I had the same problem when try to run in docker, fixed in #260
